tools/report-converter/codechecker_report_converter/report/hash.py

```diff
@@ -87,7 +87,7 @@ def __get_report_hash_path_sensitive(report: Report) -> List[str]:
 
         hash_content = [report.file.name,
                         report.checker_name,
-                        event.message,
+                        report.static_message,
```

Comment:
You documented this field for the report class, but you ough to in the function as well. It still talks about "checker message".

Reply:
I updated the documentation, thanks for the comment!